pkgs/applications/graphics/hydrus/default.nix

```diff
@@ -32,6 +33,18 @@ python3Packages.buildPythonPackage rec {
     qtcharts
   ];
 
+  desktopItem = makeDesktopItem {
```

Comment:
better to use `desktopItems` (It's a list of the "desktopItem" stuff) and add `copyDesktopItems` in `nativeBuildInputs`